frontend/src/App.js

```diff
@@ -64,13 +65,12 @@ class App extends Component {
   }
   render() {
     return (
-    //   <div className="App">
-    //     <header className="App-header"></header>
-    //     <RecipeList recipes={recipes} />
-    //   </div>
-    // );
-
-      <Router history={history}> 
+      <div className="App">
+        <header className="App-header"></header>
+        
+        <SignIn />
```

Comment:
I think we're going to want to use the router to render the signup page at a separate URL? Is this different from the route on line 94?